tsqa/conf_plugin.py

```diff
@@ -0,0 +1,28 @@
+import logging
+import os
+
+from nose.plugins import Plugin
+
+log = logging.getLogger('nose.plugins.ConfPlugin')
+
+class ConfPlugin(Plugin):
+
+    name = 'conf-plugin'
+    def options(self, parser, env=os.environ):
+        """Register commandline options.
+        """
+        Plugin.options(self, parser, env)
+        parser.add_option('--keep-tmp', action='store_true', dest='keep_tmp', default=True,
+                          help="Keep tmp files after running successfully")
+        parser.add_option('--sleep-in-sec', type='int', default=0,
+                          dest='sleep_in_sec',
+                          help='Sleep time before ATS start and after ATS stop to allow enough time for async tests')
+        parser.add_option('--standalone-ats-port', type='int', default=-1,
```

Comment:
This will probably need to follow the same server_ports markup that we have in records.config-- since ATS ports have different attributes associated with them (such as ipv4:443 -- https://docs.trafficserver.apache.org/en/latest/reference/configuration/records.config.en.html#proxy-config-http-server-ports)